src/Attestation/Attestation/Commands/AddAzAttestationPolicySigner.cs

```diff
@@ -0,0 +1,85 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.Attestation.Models;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+
+namespace Microsoft.Azure.Commands.Attestation
+{
+    /// <summary>
+    /// Set AttestationPolicy.
+    /// </summary>
+    [Cmdlet("Add", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "AttestationPolicySigner", SupportsShouldProcess = true)]
+    [OutputType(typeof(String))]
```

Comment:
Based on [cmdlet design guide line](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/design-guidelines/cmdlet-best-practices.md), the output type should never be type of String.

We encourage to have a wrapper class to encapsulate the string message

Reply:
Thanks, fixed.